README.rst

```diff
@@ -55,3 +55,9 @@ https://nbviewer.jupyter.org/github/python-visualization/folium_contrib/tree/mas
 
 
 .. |folium| image:: http://python-visualization.github.io/folium/docs-master/_images/folium_logo.jpg
+
+Contributing
+------------
+
+see `.github/CONTRIBUTING.md`
```

Comment:
Maybe we could add one or two lines here so it's a bit more welcoming. Something about how folium is maintained by volunteers, that contributions are welcome, and that people should check the issues.

Reply:
see new commits